zebra-rpc/src/config.rs

```diff
@@ -0,0 +1,25 @@
+//! User-configurable RPC parameters.
+
+use std::net::SocketAddr;
+
+use serde::{Deserialize, Serialize};
+
+/// RPC configuration section.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields, default)]
+pub struct Config {
+    /// IP address and port for the RPC server.
+    ///
+    /// Note: RPC server will not start if this is `None`.
+    ///
+    // The recommended ports for the RPC server is:
+    /// - Mainnet: 8232
+    /// - Testnet: 83232
```

Comment:
Note for me: This can't possibly be right, the highest TCP port is 65535.

Let's just follow the pattern of the other testnet ports:
```suggestion
    /// - Testnet: 18232
```

Reply:
Also we should explicitly use 127.0.0.1 here, for security reasons.